hack/verify-licenses.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: `hack/verify-licenses.sh`.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+
+
+# Ensure that we find the binaries we build before anything else.
+export GOBIN="${KUBE_OUTPUT_BINPATH}"
+PATH="${GOBIN}:${PATH}"
+
+
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
+
+
+# Color Codes
+RED="\033[0;31m"
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+CAYAN="\033[0;36m"
+GRAY="\033[0;37m"
+NC="\033[0m" # No Color
+
+
+declare -a packages_flagged
+declare -a allowed_licenses
+
+LICENSES_JSON="/tmp/$(cd /tmp && mktemp licenses_json.XXXXXX)"
+LICENSES_CSV="/tmp/$(cd /tmp && mktemp licenses_csv.XXXXXX)"
+LICENSES_DUMP="/tmp/$(cd /tmp && mktemp licenses_dump.XXXXXX)"
+
+# Install go-licenses
+printf '~%.0s' {1..120}; echo
+echo -e "\nInstalling go-licenses..."
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  go install github.com/google/go-licenses@v1.0.0
+popd >/dev/null
+
+
+# Fetching CNCF Approved List Of Licenses
+# Refer: https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md
+curl -s 'https://spdx.org/licenses/licenses.json' > "${LICENSES_JSON}"
+
+number_of_licenses=$(jq '.licenses | length' "${LICENSES_JSON}")
+loop_index_length=$(( number_of_licenses - 1 ))
+
+printf '~%.0s' {1..120}; echo
+echo -e "\nFetching current list of CNCF approved licenses..."
+printf '~%.0s' {1..120}; echo
+for index in $(seq 0 $loop_index_length);
+do
+	licenseID=$(jq ".licenses[$index] .licenseId" "${LICENSES_JSON}")
+	if [[ $(jq ".licenses[$index] .isDeprecatedLicenseId" "${LICENSES_JSON}") == false ]]
+	then
+		echo -e "Adding license ${CAYAN}$licenseID${NC}"
```

Comment:
This may be needed for debugging only, so let's not print this

Reply:
Ack. Removed echo print lines.